Nos/Views/New Note/ReplyPreview.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+
+/// This is a component of `NewNoteView` that displays a preview of the note being replied to. 
+struct ReplyPreview: View {
+    
+    var note: Event
+    @Environment(CurrentUser.self) var currentUser
+    
+    var body: some View {
+        VStack(spacing: 8) {
+            HStack {
+                AvatarView(imageUrl: note.author?.profilePhotoURL, size: 30)
+                Text(note.author?.safeName ?? "")
```

Comment:
As I'm working on refactoring our use of the `Text` we defined in `Font.swift`, I'm trying to avoid using it and instead using `PlainText` and then specifying the font with a modifier along these lines (assuming this uses ClarityCity):

`.font(.clarity(.medium, textStyle: .subheadline))`

I guess I'm just suggesting we avoid using `Text` since it's hard to know whether `SwiftUI.Text` or our func is being used. As I'm reading it here on GitHub, I couldn't tell you which is being used and, therefore, whether the font here is the system one or ClarityCity.

Reply:
Have you cases where `Text` is resolving to `SwiftUI.Text`? I thought all our uses of `Text` were properly shadowed to our version with Clarity City font, and the only time `SwiftUI.Text` was being used is if you typed `SwiftUI.Text` (which I don't think we do, we use `PlainText` in that case). `.font(.clarity(.medium, textStyle: .subheadline))` is awfully wordy and hard to remember, I'd like to avoid sprinkling those everywhere if we can.

Reply:
I thought I had found some but I don't know that I could point to specific instances. It would probably be a good idea to evaluate this and determine for sure.

I was in the process of removing our shadowed Text to improve clarity at the call site -- I thought @martindsq had suggested this and it seemed like a good idea to me, too. But let's have a discussion about that outside of this PR -- perhaps on my open PR: #963.